Opt 1849 (#2)
camunda/camunda-optimize-examples: decision-import-plugin/resolve-reference-outputs/README.md

            </goals>
            <configuration>
              <finalName>${project.artifactId}</finalName>
              <descriptorRefs>
                <descriptorRef>jar-with-dependencies</descriptorRef>
              </descriptorRefs>
            </configuration>
          </execution>
        </executions>
      </plugin>
      <plugin>
        <artifactId>maven-jar-plugin</artifactId>
        <version>3.0.2</version>
        <executions>
          <execution>
            <id>default-jar</id>
            <phase>none</phase>
          </execution>
        </executions>
      </plugin>
    </plugins>
  </build>
```

Note: Some libraries are already used in Optimize and therefore provided out of the box.
Instead of including those to an uber jar, just set the scope of those dependencies to
`provided`. To get an overview, what is already provided by Optimize, have a look at
the [third party libraries][5] in the Optimize documentation. Using your own version
of libraries that are already included in Optimize might cause weird side effects otherwise.

### Implement the decision import plugin

Let's say you have a situation where you need to use some of the Pokémon abilities.

> Depending on the ability, you need to decide which Pokémon needs to be summoned.

Reviewer: I like you description! Would just mention the term decision definition already here to give context for the next sentence. 
```suggestion
Depending on the ability, a decision definition decides which Pokémon needs to be summoned.
```

> So, as a desicion output you get an Pokémon ID, which is not really useful for the data analysis, as you want to see 

Reviewer: ```suggestion
So, as a decision output you get an Pokémon ID, which is not really useful for the data analysis, as you want to see 
```
